problems/1.types+operators.js

```diff
@@ -1,21 +1,23 @@
 /** Takes in an value and returns if it is a number
-* @param {any} val - The input value
-* @returns {boolean} - Is val a number
-*/
-function isNumber() {
-
-}
+ * @param {any} val - The input value
+ * @returns {boolean} - Is val a number
+ */
+function isNumber(num) {
+  let age = num
+    return age +7
+  }
+  console.log(typeof(isNumber(30)))
 
 /** Takes in a value and returns whether it is truthy
-* @param {any} val - The input value
-* @returns {boolean} - Is val truthy
-*/
-function isTruthy() {
-
+ * @param {any} val - The input value
+ * @returns {boolean} - Is val truthy
+ */
+function isTruthy(num) {
+  return num === num
```

Comment:
This will always return true. YOu're checking is something is equal to itself. Instead we need to be checking if it's truthy or falsey.  Remember the only falsey values in JS are 0, "",  null, undefined, NaN, false.  
YOu can use an if else to do this, or a !!